src/Symfony/Bundle/FrameworkBundle/Test/BrowserKitAssertionsTrait.php

```diff
@@ -28,23 +28,38 @@
  */
 trait BrowserKitAssertionsTrait
 {
-    public static function assertResponseIsSuccessful(string $message = '', bool $verbose = true): void
+    private static bool $defaultVerboseMode = true;
+
+    public static function setBrowserKitAssertionsAsVerbose(bool $verbose): void
     {
+        self::$defaultVerboseMode = $verbose;
+    }
+
+    public static function assertResponseIsSuccessful(string $message = '', ?bool $verbose = null): void
+    {
+        $verbose ??= self::$defaultVerboseMode;
+
         self::assertThatForResponse(new ResponseConstraint\ResponseIsSuccessful($verbose), $message);
     }
```

Comment:
```suggestion
    public static function assertResponseIsSuccessful(string $message = '', ?bool $verbose = null): void
    {
        self::assertThatForResponse(new ResponseConstraint\ResponseIsSuccessful($verbose ?? self::$defaultVerboseMode), $message);
    }
```

We could use this notation also to spare a potential assignment. Just a thought, the current code also suits me fine

Reply:
Done in https://github.com/symfony/symfony/pull/60788/commits/0e1540ecfda50366bea0488a1f5eaedeb8e7a0ea